packages/runtime/container-runtime/.eslintrc.cjs

```diff
@@ -14,6 +14,13 @@ module.exports = {
 	rules: {
 		"@typescript-eslint/strict-boolean-expressions": "off",
 		"@fluid-internal/fluid/no-unchecked-record-access": "warn",
+
+		// #region TODO:AB#3027: remove overrides and upgrade config to `recommended`
+
+		"jsdoc/multiline-blocks": ["error", { noSingleLineBlocks: true }],
+		"jsdoc/require-description": ["error", { checkConstructors: false }],
+
+		// #endregion
```

Comment:
Even with these overrides and the fixes from the PR we can't move to recommended yet?

Reply:
No. This package has thousands of violations against recommended. So I'm going to do things incrementally by rules, then toggle at the end.

Reply:
I also want to make sure each PR is reasonable to review. The amount of churn required to tackle _all_ of the rules will be substantial.